injectors/moveplugs.js

```diff
@@ -0,0 +1,11 @@
+const fse = require('fs-extra');
```

Comment:
This should be made into a migration script (src/Powercord/migrations, see existing one for info)